custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql

```diff
@@ -0,0 +1,62 @@
+DROP VIEW IF EXISTS bihar_demographics_view CASCADE;
+CREATE VIEW bihar_demographics_view AS
+    SELECT
+        "bihar_demographics"."awc_id" AS "awc_id",
+        "awc_location"."awc_name" AS "awc_name",
+        "awc_location"."awc_site_code" AS "awc_site_code",
+        "bihar_demographics"."supervisor_id" AS "supervisor_id",
+        "awc_location"."supervisor_name" AS "supervisor_name",
+        "awc_location"."supervisor_site_code" AS "supervisor_site_code",
+        "bihar_demographics"."block_id" AS "block_id",
+        "awc_location"."block_name" AS "block_name",
+        "awc_location"."block_site_code" AS "block_site_code",
+        "bihar_demographics"."district_id" AS "district_id",
+        "awc_location"."district_name" AS "district_name",
+        "awc_location"."district_site_code" AS "district_site_code",
+        "bihar_demographics"."state_id" AS "state_id",
```

Comment:
took from `bihar_demographics` table so that filter can use the index created on that table.